extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts

```diff
@@ -51,7 +53,7 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 					label: option.displayName,
 					value: option.name,
 					checked: option.displayName === defaultValue,
-					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? false : this._fieldInfo.enabled // Dynamic enablement is initially set to false
+					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? this._enabled : this._fieldInfo.enabled // Dynamic enablement is initially set to false
```

Comment:
This just needs to be `enabled: this._enabled` - the logic only needs to be done on initialization (the field info should never change for the dynamic enablement scenario)